backend/private-graph/graph/schema.resolvers.go

```diff
@@ -6027,21 +6027,6 @@ func (r *queryResolver) IsProjectIntegratedWith(ctx context.Context, integration
 		return false, e.Wrap(err, "error querying project")
 	}
 
-	if integrationType == modelInputs.IntegrationTypeClickUp {
-		var projectMapping *model.IntegrationProjectMapping
-		if err := r.DB.Where(&model.IntegrationProjectMapping{
-			ProjectID:       project.ID,
-			IntegrationType: integrationType,
-		}).First(&projectMapping).Error; err != nil {
-			return false, nil
-		}
-
-		if projectMapping == nil {
-			return false, nil
-		}
```

Comment:
This was the bug, we should have had an `else` here with `return true, nil`

Reply:
I moved the code into the `backend/integrations` directory to assist with writing a regression test.